### PR TITLE
add car parking availability

### PR DESCRIPTION
**Pull request by xlqian**
p r

### CODE REVIEW

File: js/summary.js [hunk 1]
```diff
@@ -448,6 +448,17 @@ summary.make.stands = function(context, json) {
     ));
 };
 
+summary.make.parking_places = function(context, json) {
+    return  $('<span/>').text(sprintf(
+        'available: %d, occupied: %d, available disabled: %d, occupied disabled: %d, total: %d',
+        json.available_places,
+        json.occupied_places,
+        json.available_disabled || 0,
+        json.occupied_disabled || 0,
```
Comment (TeXitoi) [inline js/summary.js]: maybe better to not print the field if it does not exists

**Comment by xlqian** [comment]
screenshot from 2017-08-11 11 49 10

File: js/summary.js [hunk 2]
```diff
@@ -447,6 +447,24 @@ summary.make.stands = function(context, json) {
         json.total_stands
     ));
 };
+    var console;
```
Comment (TeXitoi) [inline js/summary.js]: dd

File: js/summary.js [hunk 3]
```diff
@@ -448,6 +448,24 @@ summary.make.stands = function(context, json) {
...
 };
 
+summary.make.car_park = function(context, json) {
+
+    var PRM_parking = '';
+    if (json.available_PRM !== null && json.occupied_PRM !== null) {
+        PRM_parking = sprintf('available_PRM: %d, occupied_PRM: %d',
+        json.available_PRM,
+        json.occupied_PRM
+    );
```
Comment (TeXitoi) [inline js/summary.js]: strange indentation + only print each field if present as in jormungandr.

File: js/summary.js [hunk 4]
```diff
@@ -448,6 +448,18 @@ summary.make.stands = function(context, json) {
...
+summary.make.car_park = function(context, json) {
+    var park_summary = $('<span/>');
+    $.each(['available', 'occupied', 'available_PRM', 'occupied_PRM',
+            'total_places'],
+        function(_, value) {
+            if (value in json) {
+                park_summary.append(sprintf('%s: %d ', value, json[value]));
+            }
+    });
+    return park_summary;
```
Comment (TeXitoi) [inline js/summary.js]: Something like:
```js
var text = $.map(['available', 'occupied', 'available_PRM', 'occupied_PRM', 'total_places'], function(value) {
    return value in json ? sprintf('%s: %d ', value, json[value]) : null;
}).join(', ');
return $('<span>').text(text);
```

**Pull request merged**